tox.ini

```diff
@@ -230,6 +230,9 @@ ignore_missing_imports = True
 [mypy-deepdiff.*]
 ignore_missing_imports = True
 
+[mypy-twikit.*]
```

Comment:
where's this even used?

Reply:
it's used for validating the twitter cookies